packages/paste-core/components/sidebar/package.json

```diff
@@ -24,15 +24,22 @@
     "tsc": "tsc"
   },
   "peerDependencies": {
+    "@twilio-paste/anchor": "^11.0.0",
```

Comment:
we shouldn't need anchor, spinner, or stack in the package.json I don't think

Reply:
Button depends on them

Reply:
ohhhhh